frontend/src/components/layout/UserDropDown/UserDropDown.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Popper, Box, List, ListItem, ListItemText } from '@mui/material';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+import {
+  UserInfo,
+  UserAvatar,
+  UserName,
+  UserPopper,
+  ArrowDown,
+  ArrowUp,
+  DividerHorizontal,
+  Link,
+} from 'components/Layout/UserDropDown/styled';
+import { User } from 'types/user';
+
+interface PopperControl {
+  isOpen: boolean;
+  onClick: (event: React.MouseEvent<HTMLElement>) => void;
+  anchorEl: HTMLElement | null;
+}
+
+type Props = User & PopperControl;
+
+const DropDown: React.FC<Props> = ({
+  userName,
+  userRole,
+  userUnit,
+  userDepartment,
+  userGroup,
+  avatarUrl,
+  isOpen,
+  onClick,
+  anchorEl,
+}) => (
+  <Box>
+    <UserInfo onClick={(event: React.MouseEvent<HTMLElement>) => onClick(event)}>
```

Comment:
No need to use an arrow function here, the same:
```suggestion
    <UserInfo onClick={onClick}>
```